### ISSUE TITLE
Peer to peer review

### ISSUE THREAD

**Issue by michael-duke** [PRATAP-KUMAR/webpack-todolist#5]
## Feedback for Pratap

- Make sure to follow best JavaScript to extend your codebase
- It's a optional; from an accessibility stand point when user press tab the button should have an outline.
- Adjust your function that updates the DOM, and also the one that changes the complete flag so everything can be current.
- You can also move the container up and add an addtional overflow property on it so when you scroll the user can access the todos. :smile: add a max-height to it
- Make sure the todo desc is aligned next to the checkbox and not centered.
